src/main/java/net/greatstart/dto/DtoCategory.java

```diff
@@ -0,0 +1,8 @@
+package net.greatstart.dto;
+
+import lombok.Data;
+
+@Data
+public class DtoCategory {
+    private String name;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "name" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)